var/spack/repos/builtin/packages/py-jupyterhub/package.py

```diff
@@ -31,4 +36,5 @@ class PyJupyterhub(PythonPackage):
     depends_on('py-python-oauth2@1.0:', when='@:9.4', type=('build', 'run'))
     depends_on('py-pamela', type=('build', 'run'))
     depends_on('py-notebook', type=('build', 'run'))
+    depends_on('py-prometheus-client@0.4.0:', type=('build', 'run'), when='@1.4.1:')
     depends_on('py-prometheus-client@0.0.21:', type=('build', 'run'))
```

Comment:
Missing dependency on jupyter-telemetry

Reply:
`Jupyter-telemetry` doesn't yet exist in Spack, unless I am missing it. I will add it.

How did you get the list of required dependencies and their associated versions?

Reply:
From the `requirements.txt`.